extensions/dicom-microscopy/src/getCommandsModule.ts

```diff
@@ -124,15 +124,15 @@ export default function getCommandsModule({
     },
 
     incrementActiveViewport: () => {
```

Comment:
Please remove this function from microscopy - it belongs in the default one as a generic function that sets the next/previous viewport active.